File: python/exercise_119.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        a = []
        for i in range(rowIndex+1):
            for j in range(i+1):
                print(j)
                if(j == 0):
                    a.append([1])
                elif(j == i):
                    a[i].append(1)
                else:
                    a[i].append(a[i-1][j-1]+a[i-1][j])
        return a[rowIndex]

File: java/exercise_119.java
class Solution {
    public List<Integer> getRow(int numRows) {
        List<List<Integer>> pascalTriangle = new ArrayList<>();
        pascalTriangle.add(List.of(1));
        for (int rowIndex = 0; rowIndex < numRows; rowIndex++) {
            List<Integer> currentRow = new ArrayList<>();
            currentRow.add(1);
            List<Integer> previousRow = pascalTriangle.get(rowIndex);
            for (int colIndex = 1; colIndex < previousRow.size(); colIndex++) {
                int sum = previousRow.get(colIndex - 1) + previousRow.get(colIndex);
                currentRow.add(sum);
            }
            currentRow.add(1);
            pascalTriangle.add(currentRow);
        }
        return pascalTriangle.get(pascalTriangle.size() - 1);
    }
}